docs.openc3.com/docs/configuration/_command.md

```diff
@@ -1,6 +1,9 @@
 ---
 sidebar_position: 4
 title: Commands
+description: Command definition file format and keywords
+sidebar_custom_props:
+  myEmoji: 📡
```

Comment:
I always hated the icons that went on those top level collection pages. I also updated the description throughout.
<img width="982" alt="Screenshot 2024-11-06 at 1 49 01 PM" src="https://github.com/user-attachments/assets/3e9a5c77-66b9-4b25-a9e8-fbbf3738ff0f">